app/src/main/res/values/strings.xml

```diff
@@ -253,4 +265,17 @@
     <string name="rate_this_app">Rate this app</string>
     <string name="on_google_play">on Google Play</string>
     <string name="report_error">Report Error</string>
-</resources>
\ No newline at end of file
+
+    <string name="add_reminder">Add reminder</string>
+    <string name="days_before_expiry">Days before expiry :</string>
+    <string name="edit_reminder">Edit Reminder</string>
+    <string name="reminders">Reminders</string>
+    <string name="days_before_expiry_date"> days before expiry date</string>
+    <string name="set_expiry_date">Set a expiry date!</string>
+    <string name="card_expired">Card is expired!</string>
+    <string name="card_text">Card </string>
+    <string name="is_about_to_expire"> is about to expire!</string>
```

Comment:
You should use a formatted string here like this, so translators can change the word order:

https://github.com/TheLastProject/Catima/blob/5faa28a7e799ba56c531363291041836e3b3deb9/app/src/main/java/protect/card_locker/BarcodeImageWriterTask.java#L242

https://github.com/TheLastProject/Catima/blob/5e593d0dede0c1262f763af1aaedf7ef6bf453e8/app/src/main/res/values/strings.xml#L53